int32[] road
string[] roadNo
int32 len1
int32 len2
# vector<int32> road